server/app/models/person.rb

```diff
@@ -24,6 +24,7 @@ class Person < ActiveRecord::Base
   has_attached_file :image
 
   has_many :castings, dependent: :destroy
+  belongs_to :listable, polymorphic: true
```

Comment:
These should probably be named something like `list_items` since it's referring to what you'll get when you call it, not the association name